<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtSphere IDE</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QTabWidget" name="sideBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::East</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="projectExplorerTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Project Explorer</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeView">
           <property name="animated">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="projectTaskList">
        <attribute name="title">
         <string>Project Task List</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="taskListLabel">
             <property name="text">
              <string>Task List</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="newTaskButton">
             <property name="text">
              <string>New Task</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="taskListTable">
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Task</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mediaPlayerTab">
        <attribute name="title">
         <string>Media Player</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="mainAreaFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="consoleSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTabWidget" name="openFileTabs">
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="tabsClosable">
            <bool>true</bool>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTabWidget" name="infoTabs">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::South</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="consoleTab">
            <attribute name="title">
             <string>Console</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="consoleText">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="buildLogTab">
            <attribute name="title">
             <string>Build Log</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextEdit" name="buildLogText">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="newProject"/>
     <addaction name="separator"/>
     <addaction name="newPlainTextFile"/>
     <addaction name="newMap"/>
     <addaction name="actionSpriteset"/>
     <addaction name="newTileset"/>
     <addaction name="newFont"/>
     <addaction name="newWindowstyle"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionTiled_map_to_Sphere_map"/>
    </widget>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionProjectName"/>
     <addaction name="separator"/>
     <addaction name="actionFile_1"/>
     <addaction name="actionFile_2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionClose_Project"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionClose_Active_Pane"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Page"/>
    <addaction name="actionSound_Test"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionOpen_Game_Directory"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionRun_Cell"/>
    <addaction name="actionPackage_Game"/>
    <addaction name="separator"/>
    <addaction name="actionProject_Properties"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure_Engine"/>
    <addaction name="actionConfigure_QtSphere_IDE"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQSIGithub"/>
    <addaction name="actionMSGithub"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuProject"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolbarNewButton"/>
   <addaction name="toolbarOpenButton"/>
   <addaction name="toolbarSaveButton"/>
   <addaction name="separator"/>
   <addaction name="toolbarPlayGame"/>
   <addaction name="toolbarProjectProperties"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionClose_Active_Pane">
   <property name="text">
    <string>Close Active Pane</string>
   </property>
  </action>
  <action name="actionStart_Page">
   <property name="text">
    <string>Start Page</string>
   </property>
  </action>
  <action name="actionSound_Test">
   <property name="text">
    <string>Sound Test</string>
   </property>
  </action>
  <action name="action_files">
   <property name="text">
    <string>&lt;files&gt;</string>
   </property>
  </action>
  <action name="actionOpen_Game_Directory">
   <property name="text">
    <string>Open Game Directory</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionProject_Properties">
   <property name="text">
    <string>Project Properties</string>
   </property>
  </action>
  <action name="actionPackage_Game">
   <property name="text">
    <string>Package Game...</string>
   </property>
  </action>
  <action name="actionConfigure_Engine">
   <property name="text">
    <string>Configure Engine...</string>
   </property>
  </action>
  <action name="actionConfigure_QtSphere_IDE">
   <property name="text">
    <string>Configure QtSphere IDE</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="newProject">
   <property name="text">
    <string>Project...</string>
   </property>
  </action>
  <action name="newPlainTextFile">
   <property name="text">
    <string>Text/Script file</string>
   </property>
  </action>
  <action name="newMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="newTileset">
   <property name="text">
    <string>Tileset</string>
   </property>
  </action>
  <action name="newFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="newWindowstyle">
   <property name="text">
    <string>Windowstyle</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="toolbarNewButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="toolbarOpenButton">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="toolbarSaveButton">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
  </action>
  <action name="toolbarPlayGame">
   <property name="icon">
    <iconset resource="res/qsires.qrc">
     <normaloff>:/icons/sphere-icon.png</normaloff>:/icons/sphere-icon.png</iconset>
   </property>
   <property name="text">
    <string>Play Game</string>
   </property>
   <property name="toolTip">
    <string>Play Game</string>
   </property>
  </action>
  <action name="toolbarProjectProperties">
   <property name="icon">
    <iconset resource="res/qsires.qrc">
     <normaloff>:/icons/1385573251-24px.png</normaloff>:/icons/1385573251-24px.png</iconset>
   </property>
   <property name="text">
    <string>Project Properties</string>
   </property>
   <property name="toolTip">
    <string>Project Properties</string>
   </property>
  </action>
  <action name="actionContextMenuItem1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/qsires.qrc">
     <normaloff>:/icons/sphere-icon.png</normaloff>:/icons/sphere-icon.png</iconset>
   </property>
   <property name="text">
    <string>contextMenuItem1</string>
   </property>
  </action>
  <action name="actionTiled_map_to_Sphere_map">
   <property name="text">
    <string>Tiled map</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project...</string>
   </property>
  </action>
  <action name="actionProjectName">
   <property name="text">
    <string>Project Name</string>
   </property>
  </action>
  <action name="actionFile_1">
   <property name="text">
    <string>field.rmp</string>
   </property>
  </action>
  <action name="actionFile_2">
   <property name="text">
    <string>field2.rmp</string>
   </property>
  </action>
  <action name="actionRun_Cell">
   <property name="text">
    <string>Run Cell</string>
   </property>
  </action>
  <action name="actionSpriteset">
   <property name="text">
    <string>Spriteset</string>
   </property>
  </action>
  <action name="actionQSIGithub">
   <property name="text">
    <string>QtSphere IDE on GitHub</string>
   </property>
  </action>
  <action name="actionMSGithub">
   <property name="text">
    <string>miniSphere on GitHub</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/qsires.qrc"/>
 </resources>
 <connections/>
</ui>
